testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Standard.java

```diff
@@ -184,6 +185,59 @@ else if (logicalName.startsWith(WORKER_NTH)) {
         enableTrinoJavaDebugger(dockerContainer, logicalName, debugPort);
     }
 
+    private static void enableTrinoJmxRmi(DockerContainer dockerContainer)
+    {
+        String logicalName = dockerContainer.getLogicalName();
+
+        int debugPort;
+        if (logicalName.equals(COORDINATOR)) {
+            debugPort = 6005;
+        }
+        else if (logicalName.equals(WORKER)) {
+            debugPort = 6009;
+        }
+        else if (logicalName.startsWith(WORKER_NTH)) {
+            int workerNumber = parseInt(logicalName.substring(WORKER_NTH.length()));
+            debugPort = 6008 + workerNumber;
+        }
+        else {
+            throw new IllegalStateException("Cannot enable Java debugger for: " + logicalName);
+        }
+
+        enableTrinoJmxRmi(dockerContainer, logicalName, debugPort);
+    }
+
+    private static void enableTrinoJmxRmi(DockerContainer container, String containerName, int jmxPort)
+    {
+        log.info("Enabling Java JMX RMI for container: '%s' on port %d", containerName, jmxPort);
+
+        try {
+            FileAttribute<Set<PosixFilePermission>> rwx = PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwxrwxrwx"));
+            Path script = Files.createTempFile("enable-java-jmx-rmi", ".sh", rwx);
+            script.toFile().deleteOnExit();
+            Files.writeString(
+                    script,
+                    format(
+                            "#!/bin/bash\n" +
+                                    "printf '%%s\\n' '%s' >> '%s'\n",
+                                    "-Dcom.sun.management.jmxremote=true\n" +
```

Comment:
It's not clear that this is an argument and not part of the template, can you create a variable for it?

Also, since we're on JDK 17, maybe use a multiline strings.